packages/docs/.vitepress/components/VueMasteryLogoLink.vue

```diff
@@ -0,0 +1,84 @@
+<script setup lang="ts">
+const props = defineProps<{
+  for: string,
```

Comment:
let's just use a boolean
```suggestion
  description?: boolean,
```